spec/models/ncr/work_order_spec.rb

```diff
@@ -453,4 +453,19 @@
       expect(wo.building_id).to be_nil
     end
   end
+
+  describe "#current_approver_email_address" do
+    it "returns the first pending approval's email address" do
+      wo = FactoryGirl.create(:ncr_work_order, :with_approvers)
```

Comment:
Might be outside the scope of this PR, but what do you think about include FactoryGirl syntax methods so this can just be `wo = create(:ncr_work_order, :with_approvers)` ? Would be a lot less typing!!

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#configure-your-test-suite